testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java

```diff
@@ -304,7 +304,13 @@ public void testShowSchemasLikeWithEscape()
             Set<Object> allSchemas = computeActual("SHOW SCHEMAS").getOnlyColumnAsSet();
             assertEquals(allSchemas, computeActual("SHOW SCHEMAS LIKE '%_%'").getOnlyColumnAsSet());
             Set<Object> result = computeActual("SHOW SCHEMAS LIKE '%$_%' ESCAPE '$'").getOnlyColumnAsSet();
-            assertNotEquals(allSchemas, result);
+            Set<Object> schemasWithoutUnderscore = computeActual("SHOW SCHEMAS NOT LIKE '%_%'").getOnlyColumnAsSet();
```

Comment:
`SHOW SCHEMAS` doesn't support `NOT` option in my understanding.
```
    | SHOW SCHEMAS ((FROM | IN) identifier)?
        (LIKE pattern=string (ESCAPE escape=string)?)?                 #showSchemas
```

Reply:
Adjusted to check the schema names from allSchemas instead. PTAL